infra/terraform/platform/main.tf

```diff
@@ -205,3 +205,27 @@ module "buckets_archiver" {
     "component", "buckets-archiver",
   ), var.tags)}"
 }
+
+module "kubernetes_master_monitoring" {
+  source = "../modules/elb_cloudwatch_alerts"
+
+  name          = "${terraform.workspace}-kubernetes-master-alerts"
+  elb_name      = "${var.kubernetes_master_elb_name}"
+  alarm_actions = ["${module.softnas_monitoring.stack_notifications_arn}"]
```

Comment:
Should the alarms for the masters go into the SoftNAS topic?

Reply:
Yeah I think I'll probably eventually take the SNS notification stuff out of the SoftNAS module as it probably doesn't make sense for us to set up and new SNS topic for each set of alerts.

Reply:
I've create a new SNS topic for AP infra alerts and pointed the new alerts at this topic rather then reusing the softnas topic